depends/packages/fontconfig.mk

```diff
@@ -11,6 +11,7 @@ define $(package)_set_vars
 endef
 
 define $(package)_config_cmds
+  export PKG_CONFIG_PATH=$(host_prefix)/lib/pkgconfig && \
```

Comment:
@josephbisch Sorry for the delay.

I think I see. Their pkg-config wrapper script is kinda annoying, I wonder why they didn't build one suited for the environment rather than just wrapping (what i assume is) a default build.

The issue is here: https://github.com/raspberrypi/tools/blob/master/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/arm-linux-gnueabihf-pkg-config

That will override what we've already set, but I'm not sure why your changes manage to re-override theirs.

I suspect that you could fix this locally in your environment instead by doing:
```export PKG_CONFIG=arm-linux-gnueabihf-pkg-config-real``` before building depends.